src/expressions/if-expr.md

```diff
@@ -41,12 +41,15 @@ assert_eq!(y, "Bigger");
 
 > **<sup>Syntax</sup>**\
 > _IfLetExpression_ :\
-> &nbsp;&nbsp; `if` `let` [_Pattern_] `=` [_Expression_]<sub>_except struct or lazy boolean operator expression_</sub>
+> &nbsp;&nbsp; `if` `let` [_Pattern_] `=` _Scrutinee_
```

Comment:
Instead of re-defining it, can you link to a single definition?  Something like:

```suggestion
> &nbsp;&nbsp; `if` `let` [_Pattern_] `=` [_Scrutinee_]
```

With a link to the `match` section, like:

```md
[_Scrutinee_]: match-expr.md
```